CalendarApp/BookingRoom/Filters/GoogleExceptionFilterAttribute.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using System.Web.Http.Filters;
+using BookingRoom.Helpers;
+using Google;
+using log4net;
+
+namespace BookingRoom.Filters
+{
+    public class GoogleExceptionFilterAttribute : ExceptionFilterAttribute
+    {
+        private readonly ILog _log =
+            LogManager.GetLogger(SettingsHelper.GetLoggerName());
+
+
+        public override void OnException(HttpActionExecutedContext actionExecutedContext)
+        {
+            if (actionExecutedContext.Exception is GoogleApiException)
+            {
+                actionExecutedContext.Response = new HttpResponseMessage
+                {
+                    Content = new StringContent("Something gone wrong with input data"),
```

Comment:
the first part of exception message will be:
The service calendar has thrown an exception: Google.GoogleApiException: Google.Apis.Requests.RequestError
Not Found [404]
Errors [
	Message[Not Found] Location[ - ] Reason[notFound] Domain[global]
]